src/calculateRentalCost.js

```diff
@@ -29,6 +29,18 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const cost = days * 40;
+  let result = 0;
+
+  if (days >= 7) {
+    result = cost - 50;
+  } else if (days >= 3) {
+    result = cost - 20;
+  } else {
+    result = cost;
+  }
```

Comment:
Hi! Please, take a look at the checklist.
the result should be returned immediatly in ```if``` statement.
![image](https://user-images.githubusercontent.com/127268424/236833574-aee913c6-9c9b-41c1-9f0e-5bd49d01731f.png)